packages/core/src/views/connect/Index.svelte

```diff
@@ -356,19 +373,23 @@
 <style>
   .container {
     font-family: var(--onboard-font-family-normal, var(--font-family-normal));
-    line-height: 24px;
-    color: var(--onboard-gray-700, var(--gray-700));
     font-size: var(--onboard-font-size-5, var(--font-size-5));
-    height: var(--onboard-connect-content-height, 440px);
-    overflow: hidden;
+    color: var(--onboard-gray-700, var(--gray-700));
     background: var(
       --onboard-main-scroll-container-background,
       var(--onboard-white, var(--white))
     );
+    line-height: 24px;
+    margin-bottom: 2rem;
+    overflow: hidden;
+    position: relative;
+    display: flex;
+    height: min-content;
+    flex-flow: column-reverse;
   }
 
   .content {
-    width: var(--onboard-connect-content-width, 488px);
+    width: 100%;
```

Comment:
@taylorjdawson I think this was a community request - 100% could be the fallback instead of 488px

Reply:
Found it below, nps!

Reply:
Oh okay good idea 👍🏾